### DISCUSSION TITLE
Two repositories

### DISCUSSION THREAD

**Discussion by drakkai** [fluxcd/flux2#2659] [Q&A] [answered]
Hello, I have two git repositories for my solution: one for flux-system and all Kubernetes manifests (yaml files with kustomization, deployments, services etc.) and second for the website content. Is it possible to configure flux to recreate specific pod if there are new changes in the second repository (this one without kubernetes resources)?

**Comment by kingdonb** [comment, marked as answer]
Flux's automation's design is to pull when YAML resources are updated or container images are being created.

If you have a design where you just need `rollout restart` for some pod to execute whenever a repository without any YAML resources in it gets updated, you can configure that repository to push an update to some configmap with a hash of the content that you care about when it changes. Then embed that hash with a [variable substitution](https://fluxcd.io/docs/components/kustomize/kustomization/#variable-substitution) in the `Deployment.spec.pod.spec.metadata.annotations` so that a new value in the configmap results in a new rollout for the deployment.

The scheme I'm describing is approximately what's described here, but with less: https://fluxcd.io/docs/guides/helmreleases/#refer-to-values-in-configmaps-generated-with-kustomize

You'll want to follow that example in order to get the whole idea. It can be applied wholesale as in the example if you're already using Helm, or it can be adapted according to the structure I described if you need it to work without Helm.
